### DISCUSSION TITLE
Run errors

### DISCUSSION THREAD

**Discussion by ASKapat** [shannon-lab/zapdos#171] [Q&A Software Development] [answered]
Hi, I recently updated Crane and Zapdos and since then I receive "Unused parameter" (I've gotten it for use_ad in the DriftDiffusionAction, as well  potential, position_units in various BCs if that matters/helps) when trying to run inputs that previously worked.

**Comment by csdechant** [comment]
This is a new error message that is triggered when the user tries to add an undefined parameter to an object. For example, `DriftDiffusionAction` does not have a `use_ad` parameter. So if a user adds `use_ad = true` to the `DriftDiffusionAction` block, Zapdos will give you this error.

**Reply by ASKapat** [reply]
Removing it worked, thanks!

**Comment by cticenhour** [comment, marked as answer]
MOOSE updated in December to change the default for how unused parameters were treated (previously as a warning, now as an error) during testing. Inputs housed in Zapdos were fixed up, but if you are testing custom inputs you might have some parameters still hanging aorund

`use_ad` was removed from DriftDiffusionAction back in #80, so the fix for that and others is to simply remove or comment out those parameters in your input files.

**Reply by ASKapat** [reply]
Removing them worked, thanks!
